docs/source/release_notes/release-notes.rst

```diff
@@ -42,6 +42,15 @@ develop
     *    - Type
          - Change
 
+    *    - |devbreak|
+         - The implementation of the TimeLock server has been separated into a new project, `timelock-impl`.
```

Comment:
Yea at this point I would not consider this a devbreak and just a change in devex